app/helpers/white_card_helpers.rb

```diff
@@ -0,0 +1,3 @@
+def users_white_cards
```

Comment:
I'd remove this helper and just use `current_user.cards.white` where cards is an association on `User` and `white` is a scope on `Card`